modules/nodes-containers-projected-volumes-about.adoc

```diff
@@ -0,0 +1,169 @@
+
+* *ConfigMap + Secrets.* Projected volumes allow you to deploy containers involving configuration data and passwords.
+For example, you might execute an Ansible playbook stored as a configmap, with some sensitive encrypted tasks that are decrypted using a vault password file.
+
+* *ConfigMap + Downward API.* Projected volumes allow you to generate a config including the pod name (available via the `metadata.name` selector). This application can then pass the pod name along with requests in order to easily determine the source without using IP tracking.
+
+* *Secrets + Downward API.* Projected volumes allow you to use a secret as a public key to encrypt the namespace of the pod (available via the `metadata.namespace` selector).
+This example allows the operator to use the application to deliver the namespace information securely without using an encrypted transport.
+
+The following are examples of pod specifications for creating projected volumes.
+
+.Pod with a secret, a downward API, and a configmap
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+  - name: container-test
+    image: busybox
+    volumeMounts: <1>
+    - name: all-in-one
+      mountPath: "/projected-volume"<2>
+      readOnly: true <3>
+  volumes: <4>
+  - name: all-in-one <5>
+    projected:
+      defaultMode: 0400 <6>
+      sources:
+      - secret:
+          name: mysecret <7>
+          items:
+            - key: username
+              path: my-group/my-username <8>
+      - downwardAPI: <9>
+          items:
+            - path: "labels"
+              fieldRef:
+                fieldPath: metadata.labels
+            - path: "cpu_limit"
+              resourceFieldRef:
+                containerName: container-test
+                resource: limits.cpu
+      - configMap: <10>
+          name: myconfigmap
+          items:
+            - key: config
+              path: my-group/my-config
+              mode: 0777 <11>
+----
+
+<1> Add a `volumeMounts` section for each container that needs the secret.
+<2> Specify a path to an unused directory where the secret will appear.
+<3> Set `readOnly` to `true`.
+<4> Add a `volumes` block to list each projected volume source.
+<5> Specify any name for the volume.
+<6> Set the execute permission on the files.
+<7> Add a secret. Enter the name of the secret object. Each secret you want to use must be listed.
+<8> Specify the path to the secrets file under the `mountPath`. Here, the secrets file is in *_/projected-volume/my-group/my-config_*.
+<9> Add a Downward API source.
+<10> Add a ConfigMap source.
+<11> Set the mode for the specific projection
+
+[NOTE]
+====
+If there are multiple containers in the pod, each container needs a `volumeMounts` section, but only one `volumes` section is needed.
+====
+
+
+.Pod with multiple secrets with a non-default permission mode set
```

Comment:
Do you need two sample yaml files in the concept? I think you should remove one (or both) of them to separate reference topics.